### PR TITLE
Ms photogram industrial 2

### PR DESCRIPTION
**Pull request by MayaS1111**

### CODE REVIEW

File: app/views/layouts/application.html.erb
Comment (thierrychau) [inline app/views/layouts/application.html.erb:1]: These should not be part of this pull request, please remove these.

While having these is actually good in the bigger picture, it's good practice to keep pull requests very focused on what is asked of you, especially when contributing to larger code where other changes may need some other approval. In addition, keeping pull request focused on one feature helps tremendously with code review, as you can imagine with 60 files to review.

**Review by thierrychau — CHANGES_REQUESTED** [review]
Only one minor but important comment, please see below. 

My request might look very odd and nitpicky, and it is, but we want to make sure each pull request stays concise and focused on the task.

In a team, it is very possible that the flash alerts would be its own issue and pull request, and someone else might be responsible for it.

File: app/views/layouts/application.html.erb
Comment (MayaS1111) [reply app/views/layouts/application.html.erb:1]: Thanks, I will fix that now. I didn't know; I'll keep that in mind for now.

Comment (MayaS1111) [reply app/views/layouts/application.html.erb:1]: Done! They are removed as a new commit.

**Review by thierrychau — APPROVED** [review]
Great job, good to merge. Thanks for the edits!

**Comment by reherr** [comment]
nice job!

**Pull request merged**